frontend/src/components/OurProjects.vue

```diff
@@ -78,7 +225,7 @@ export default {
   height: 100%;
 }
 
-#hoverd-img {
+#card--hover {
```

Comment:
Do we have a standard with two dashes in `id` name? Also, a fun fact is that we shall not introduce multiple ids on the same page (as that's not supported by browsers) and I see that you're using that id in `v-for` loop.

Reply:
In this case yes. According to [BEM](http://getbem.com/naming/) two hyphens signify block modyfier as #card--hover modifies #card. The problem is rather that they should be a .class rather than #id as you've suggested.